themes/V3/5ePHB/style.less

```diff
@@ -796,6 +796,28 @@
 // *****************************
 // *      TABLE OF CONTENTS
 // *****************************/
+
+// Default Exclusions
+.page:has(.frontCover), 
+.page:has(.backCover), 
+.page:has(.insideCover), 
+.page:has(.partCover), 
+.monster,
+.noToC,
+.toc { --TOC: exclude; }
+
+// Manual Inclusion classes
+
+.addToC,
+.addh1, .addh2, .addh3, .addh4, .addh5, .addh6, .addh1h2 h1, .addh1h2 h2,
```

Comment:
Why so many classes here that all do the same thing?

Reply:
Individual and group toggles, per Header.

I don't think they work like I had in my head. I'm gonna slim this down and assume we will document how to build custom styles to work as desired.